src/main/resources/db/changelog/2024-10-15-add-label-to-homework-question-table.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - preConditions:
+      onFail: HALT
+      onError: HALT
+  - changeSet:
+      id: add-column-to-homework-question
+      author: wanderson
+      changes:
+        - sql:
+            dbms: 'h2, mysql'
+            sql: alter table homework_question add `label` varchar(36) default NULL after `question`;
```

Comment:
Probably too late for your local changelog checksums but note your syntax here. keywords should be CAPITALIZED